bot_file.py

```diff
@@ -40,6 +42,22 @@ def __init__(self, *args, **kwargs):
 
         self.known_names = KnownNames()
         self.known_names.load_from_file(self.known_names_file)
+        self.notify = Notification()
+        self.notify.load_from_file(self.notify_file)
+
+
+    def _actually_parse_message(self, message):
+        GenericSlackBot._actually_parse_message(self, message)
+
+        who_wants_it = None
+        try:
+            who_wants_it = self.notify.who_wants_it(message['text'])
+        except Exception as e:
```

Comment:
:art: do we expect `who_wants_it` to raise an exception? maybe lift it up inside the loop in GenericBot.parse_messages